apps/cms/lib/api/static.ex

```diff
@@ -417,7 +417,11 @@ defmodule CMS.API.Static do
     end
   end
 
-  def view("/admin/content/paragraphs/" <> _paragraph_id, _) do
```

Comment:
Do you no longer need this generic case for a paragraph by node_id?

Reply:
So, you can still call a paragraph by ID, but you have to include that full path ("admin/content/etc/**25**"). True, we lose the ability to simply call `paragraph(25)` but now all paragraphs have fully-adjustable aliases like pages do, so it will nearly always redirect.

@ryan-mahoney , do you think we should keep/continue to support the "permalink" style call of `paragraph(25)` along with the wildcard alias?

Reply:
if we were using that frequently I would say it might be nice to keep it around for simplicity... but since we are not, I think it's okay to do away w/ it.